src/main/java/com/rte_france/trm_algorithm/operational_conditions_aligners/HvdcAcModelingEnvironment.java

```diff
@@ -54,4 +62,56 @@ public void align(Network referenceNetwork, Network marketBasedNetwork) {
         operationalConditionAligner.align(referenceNetwork, marketBasedNetwork);
         replaceHvdcByEquivalentModel(referenceNetwork, marketBasedNetwork);
     }
+
+    List<HvdcInformation> getHvdcInformationFromNetwork(Network network) {
+        List<HvdcInformation> hvdcInformationList = new ArrayList<>();
+        SwePreprocessorParameters params = JsonSwePreprocessorImporter.read(getClass().getResourceAsStream("/hvdc/SwePreprocessorParameters.json"));
+
+        List<HvdcCreationParameters> sortedHvdcCreationParameters = params.getHvdcCreationParametersSet().stream()
+                .sorted(Comparator.comparing(HvdcCreationParameters::getId)).toList();
```

Comment:
Why reading a SWE file to extract its `HvdcCreationParameters` ?  
I think this import is too much specialized on the SWE region.
Why did you not use the `creationParametersSet` provided in the `HvdcAcModelingEnvironment` object ?